frappe/model/naming.py

```diff
@@ -14,6 +13,8 @@
 	from frappe.model.meta import Meta
 
 
+autoincremented_site_status_map = {}
```

Comment:
global variable?

Reply:
yep ..tried the `frappe.local` objects but they seem to get reset on every request. Another option is to use redis but since this will be used heavily, just used a simple dictionary.

It's basically used to store the status of site, whether `log_types` are autoincremented for the site or not.

Open to any other suggestions.